tests/analysis_test.py

```diff
@@ -60,3 +65,16 @@ def get_cmd_output(self, exec_name, options=''):
                              options, self.trace_writer.trace_root)
 
         return subprocess.getoutput(cmd)
+
+    def get_output(self, name, output):
```

Comment:
I find this method name unclear or misleading, I'd rather name it something like `save_test_result`.